socketio/src/asynchronous/client/builder.rs

```diff
@@ -97,9 +99,15 @@ impl ClientBuilder {
             max_reconnect_attempts: None,
             reconnect_delay_min: 1000,
             reconnect_delay_max: 5000,
+            data: None,
         }
     }
 
+    pub fn data<D: std::any::Any + Send + Sync>(mut self, data: Arc<D>) -> Self {
```

Comment:
Please add documentation for the method. IMO this is the perfect place for explaining what data actually is.

Reply:
Yeah! Sure, I mainly forgot to add documentation here honestly.